groups/infrastructure/data.tf

```diff
@@ -0,0 +1,38 @@
+data "vault_generic_secret" "secrets" {
+  path = "applications/${var.aws_profile}/${var.environment}/${local.stack_fullname}"
+}
+
+data "aws_kms_key" "stack_configs" {
+  key_id = "alias/${var.aws_profile}/${local.kms_key_alias}"
+}
```

Comment:
This data lookup doesn't appear to be used - is it required?

Reply:
This is fixed now. Commit is pushed.